### DISCUSSION TITLE
Remote delete doesn't work

### DISCUSSION THREAD

**Discussion by 99-lives** [remotely-save/remotely-save#150] [Q&A]
Hi team,

I'm trying to use this plugin to sync my vault between my Nextcloud server (Webdav) and my android phone, which uses remotely-save. I also have my desktop computer, where I sync my vault to the Nextcloud server through the Nextcloud desktop client, _not_ through remotely-save.

This appears to cause some issues. If I delete a note in my vault on my desktop, the deletion gets synced to my Nextcloud server not problem. But then when I sync my vault on my phone with remotely-save, the plugin uploads and recreates the deleted file to the remote server, instead of deleting the local copy, as I would expect it to.

Is this expected behavior? Or is the expectation that all Obsidian clients that use the vault should sync it with this plugin? Is this something that can be fixed?

Cheers

**Comment by xila76** [comment]
Exactly same problem and configuration here between obsidian on android with remotely save and obsidian on Mac os synced over nextcloud desktop client for mac.
Have you found any solution? Did you tried on desktop with remotely save too ? Is this solution work?

**Reply by 99-lives** [reply]
I'm using Syncthing for syncing my notes between phone and desktop. My desktop then syncs it to Nextcloud. Not using this plugin anymore.

**Reply by xila76** [reply]
Ok thanks for reply I will check this solution. I don't use syncthing at this time, have you a solution for iPad os too ? I haven't see any client for syncthing in AppStore...

**Reply by 99-lives** [reply]
I don't use anything Apple, soz

**Reply by berot3** [reply]
> Ok thanks for reply I will check this solution. I don't use syncthing at this time, have you a solution for iPad os too ? I haven't see any client for syncthing in AppStore...

There is Möbius experimental, but it costs money and I haven't tried it myself.

**Comment by frenksisco** [comment]
Similar problem. I use remotely-save to sync with same voult in Onedrive on both Obsidian desktop and mobile versions. Active synchronization on both of the .obsidian folder because I want the exact same configurations on both systems. Unfortunately, however, if I uninstall and delete plugins from the mobile version, as soon as I run sync they go back to being installed again. Some Onedrive files seem to be considered the most recent regardless of whether they were more recently modified on mobile.
